src/messages/create/signature.ts

```diff
@@ -16,8 +16,12 @@ export async function SignAndBroadcast(configuration: SignAndBroadcastConfigurat
 }
 
 async function Broadcast(configuration: BroadcastConfiguration) {
```

Comment:
Can you convert this to a `try/catch` instead of mixing async/await syntax with promises?